roles/aap_setup_install/tasks/fixes/aap_4555.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Fix for https://issues.redhat.com/browse/AAP-4555 - it'll move away the
+# setup.log before starting the installation
+
+- name: AAP_4555 | check if setup.log does exist
+  ansible.builtin.stat:
+    path: "{{ aap_setup_inst_setup_dir }}/setup.log"
+  register: __aap_setup_inst_log
+- name: AAP_4555 | create a backup of setup.log if it exists
```

Comment:
Could you please put line breaks between each task. I'm not sure why the linter hasn't picked up on it here

Reply:
ok, apparently ansible-lint doesn't check for that. Who knew...?

Either way, if you could add it, it makes it look nicer :)

Reply:
I agree, my oversight.